### DISCUSSION TITLE
support qnx710?

### DISCUSSION THREAD

**Discussion by shanHaiJing2024** [NVIDIA/stdexec#1447] [Q&A]
plans to support qnx710?

**Comment by maikel** [comment]
For large parts stdexex doesn’t depend on many QNX specific details but afaik QNX7.1 ships with a g++-8, no? I don’t know whether @ericniebler plans to ever support older compilers. 

Afaik, QNX 8 ships with g++12. that is more likely to work

**Comment by ericniebler** [comment]
i do not plan to support gcc prior to gcc-11. i would, however, accept patches.
